operators/api/v1alpha2/template_types.go

```diff
@@ -122,6 +122,10 @@ type Environment struct {
 
 	// Name of the storage class to be used for the persistent volume (when needed)
 	StorageClassName string `json:"storageClassName,omitempty"`
+
+	// +kubebuilder:default=true
+	// Whether the instance has to have the user's personal NFS volume
+	MountPersonalVolume bool `json:"mountPersonalVolume"`
```

Comment:
```suggestion
	MountMyDriveVolume bool `json:"mountMyDriveVolume"`
```

Wondering whether this naming would be clearer. @QcFe WDYT?

Reply:
Changed